applications/CoSimulationApplication/python_scripts/convergence_accelerators/aitken.py

```diff
@@ -48,33 +50,43 @@ def UpdateSolution( self, r, x ):
         if k == 0:
```

Comment:
the problem was that this condition was only fulfilled in the very first iteration, and not in every timestep

@AndreasWinterstein this will probably affect our results, but I think they should become more stable